specification/ingest/put_pipeline/PutPipelineRequestExample2.yaml

```
@@ -0,0 +1,11 @@
+summary: Creates an ingest pipeline.
```

```suggestion
summary: Create an ingest pipeline with metadata.
```

Since there are two examples for this operation, the summaries will appear in a drop-down selector and should help clarify how they're different.